packages/edit-site/src/components/editor/index.js

```diff
@@ -209,6 +209,17 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 		duration: disableMotion ? 0 : 0.2,
 	};
 
+	const isGlobalStylesPreview = path === '/wp_global_styles';
+
+	const _settings = useMemo( () => {
+		return {
+			...settings,
+			defaultRenderingMode: isGlobalStylesPreview
+				? 'template-locked'
```

Comment:
Template locked is a mode where you can edit the post things but not the template, so it's not the right thing here. For instance if you have a static page as front page, you'll still be able to edit its content.

I think instead of changing the rendering mode which is not meant to "lock" editing, we should just make sure that when isPreviewMode is on, you can't edit anything in the canvas. Maybe wrap the whole thing in `inert=true` or something (we can check how we do it in "view" mode)